content/blog/2023/09/25/2023-09-25-building-jenkinsio-with-alternative-tools.adoc

```diff
@@ -0,0 +1,83 @@
+:author: vandit1604
+:sig: gsoc
+description: >
+  The Building Jenkins.io with Alternative Tools is a project under Google Summer of Code (GSoC) 2023.
+opengraph:
+  image: /images/gsoc/opengraph.png
+links:
+  gitter: https://matrix.to/#/#jenkinsci_gsoc-2023-building-jenkinsio:matrix.org
+---
+
+This blog showcases all the work done in the project link:/projects/gsoc/2023/projects/alternative-jenkinsio-build-tool/[Building Jenkins.io with Alternative Tools] during Google Summer of Code 2023.
+
+== Table of Content
+
+. <<About Project>>
+. Coding <<Phase 1>>
+. Coding <<Phase 2>>
+. <<Future Improvements>>
+. <<Acknowledgments>>
+. <<Useful Links>>
+
+== About Project
+
+The link:/[Jenkins documentation site] is generated as a static website using Awestruct from AsciiDoc sources.
+One of the drawbacks of the current build method is that the technical documentation is not versioned.
+It is thus not possible to view the documentation for a given Jenkins version.
+Only the latest can be viewed.
+This can lead to unnecessary confusion and is a worse experience than many other documentation sites like the link:https://git-scm.com/docs/git[git site], link:https://docs.python.org/3.11/[Python[, and others.
+Another drawback of Awestruct is that it is greatly under maintained in the last two years and people from jenkins-infra are currently maintaining it.
+Relying on awestruct is not a good thing in the long run. 
+
+The preferred set of tools that will replace Awestruct are https://antora.org/[Antora] for it's out of the box versioning functionality and https://www.gatsbyjs.com/[Gatsby] for it's super fast builds.
+
+The migration was performed in the following 2 coding phases:
+
+== Phase 1
+
+In this phase, the following tasks were completed:
+
+* I used Antora to migrate the User documentation which was versioned and Developer documentation which was non-versioned while keeping all the functionalities that we have on jenkins.io right now.
+* I migrated Tutorials, Guides, Solution Pages which were all non-versioned.
+* I fixed Interpage Linking across all the pages.
+
+
+You can watch me talk about the project in the link:https://youtu.be/W4eSVCTmqb8?t=1841[Midterm Presentation here].
+
+== Phase 2
+
+In this phase, the following tasks were completed:
+
+* I used link:https://strapi.io/[Strapi CMS] with Gatsby. I would also have liked to integrate Git Workflow here which was not suggested by mentors. Hence, It was not the perfect fit for Jenkins.
+* I then moved to Gatsby and developed the blogs feature.
+* I developed other pages like LTS Changelog, LTS Weekly, Download page and many more.
+
+Here's the link to the link:https://youtu.be/M9_HPQwetMg?t=1153[Final Presentation] where you can watch me talk about the project's final phase.
+
+== Future Improvements
+
+- Some pages like Changelogs, Pipeline step reference in documentation still need more refining and are missing some features that exist on the current documentation pages.
```

Comment:
Since there are only two items in the list, we can join them with "and".
```suggestion
- Some pages like changelogs and the Pipeline steps reference still need more refining and are missing some features that exist on the current documentation pages.
```